examples/MultiStep_Griffin-BISON-Sockeye_MR/README.md

```diff
@@ -0,0 +1,42 @@
+# MultiStep_Griffin-BISON-Sockeye_MR
+
+## Purpose
+
+This example provides a demonstration on how to use WATTS to model multi-step Workflows for multiphysics simulation involving various MOOSE applications coupled by MOOSE's MultiApps system.
+
+## Code(s)
+ 
+- MOOSE MultiApp
+- Sockeye
+- BISON
+- Griffin
+
+## Keywords
+ 
+- Multi-steps
+- MultiApp
+- Micro-reactor
+- Steady-state
+- Transiens
+
+## File descriptions
+
```

Comment:
@miaoyinb - please review carefully this README. For instance, I am not sure why we need two mesh files in this repo, and I cannot find the ISOXML file.

Reply:
BISON and Griffin use different mesh. Griffin cannot handle voids in mesh but BISON needs the HP regions to be void.

The ISOXML file exist in the current `development` branch (https://github.com/watts-dev/watts/blob/development/examples/example6_adv_multiapps/unitcell_nogap_hom_xml_G11_df_MP.xml)